xtb thermo submodule unable to read hessian file from DFTB+

**Describe the bug**
A clear and concise description of what the bug is.

I am trying to compute thermochemical properties using the thermo submodule; however, the program is not able to read the `hessian.out` file from DFTB+.

My `xtb` command is as follows,

`xtb thermo --sthr 100 --temp 298.15 geo_end_final.gen -dftb+ hessian.out`

I am getting the following error,

```
[ERROR] Program stopped due to fatal error
-2- Could not read hessian from 'hessian.out'
-1- io_reader_readHessian: Could not find $hessian data group
```

--- a/src/prog/thermo.f90
+++ b/src/prog/thermo.f90
@@ -204,7 +204,7 @@ subroutine parseArguments(env, args, htype, massWeighted)
          call thermoHelp(env%unit)
          call terminate(0)
 
-      case('--dftb+')
+      case('--dftb+','--dftbplus')
          htype = hessType%dftbplus
 
       case('--turbomole')
@@ -309,7 +309,7 @@ subroutine thermoHelp(unit)
    "   --temp REAL[,...]   Temperature for thermodynamic functions in K,",&
    "                       takes a comma separated list of temperatures",&
    "",&
-   "   --dftb+             Read a DFTB+ hessian.out file, implies projection",&
+   "   --dftbplus             Read a DFTB+ hessian.out file, implies projection",&
    "",&
    "   --turbomole         Read a Turbomole Hessian file",&
    "                       use this only when $nomw is not present in control",&

